package.json

```diff
@@ -37,15 +37,8 @@
   },
   "dependencies": {
     "jws": "^3.2.2",
-    "lodash.includes": "^4.3.0",
-    "lodash.isboolean": "^3.0.3",
-    "lodash.isinteger": "^4.0.4",
-    "lodash.isnumber": "^3.0.3",
-    "lodash.isplainobject": "^4.0.6",
-    "lodash.isstring": "^4.0.1",
     "lodash.once": "^4.0.0",
```

Comment:
Per function packages are gonna be removed in lodash v5. Better use full lodash package or not use lodash at all :)